en/editor/preferences/index.md

```diff
@@ -0,0 +1,81 @@
+
+- **Spin step**: In the Properties, all numeric property has a set of UP and DOWN arrows next to the input box, values that can be used to step Input:
+
+    ![step button](https://docs.cocos.com/creator/manual/zh/getting-started/basics/editor-panels/preferences/step-button.png)
+
+    When hovering close to the value of the property name, the cursor will turn into ![mouse cursor](https://docs.cocos.com/creator/manual/zh/getting-started/basics/editor-panels/preferences/mouse-cursor.jpg) such a shape, then Drag the mouse, can also according to a continuous increase or decrease the value of step amplitude.
+
+    The above two ways to modify the value，the default step is **0.1**, spin step says each time you click the step button or drag the mouse to change the step amplitude. For example, if you mainly use numbers to integers in the script, you can take the step size is set to **1**, can be adjusted more easily.
+
+## Native Develop
+
+![native](./index/native.jpg)
+
+This page is used to set the path of the development environment required when the game is published to the native platform (iOS, Android, Mac, Windows).
+
+- **Use Builtin JS Engine**: Whether to use the engine path that comes with the Cocos Creator 3D installation path as the JavaScript engine path.This engine is used for scene rendering in the scene panel of editor, declaration of built-in components, and engine modules in other Web environments such as `preview` or `publish`.
+
+- **Custom JS Engine**: In addition to using the built-in engine, you can also go to [engine repository](https://github.com/cocos-creator/engine) to clone or fork a copy of the engine to the local anywhere, then uncheck `Use Builtin JS Engine`, then set the JavaScript engine path to your customized engine path. You can use the editor to customize your engine.
+
+- **WeChatGame APP Path**: Set the WechatGame App Path, See [WeChat Mini Games](../publish/publish-wechatgame.md).
+
+- **Android NDK Path**: Set the NDK Path, See [Setup Native Development Environment](../publish/setup-native-development.md).
+
+- **Android SDK Path**: Set Android SDK Path, See [Setup Native Development Environment](../publish/setup-native-development.md).
+
+## Data Editor
+
+This category is used to set the default open mode for resources.
+
+![edit](./index/edit.jpg)
+
+- **Script Editor**：You can use the executable file of any external text editing tool as the opening method when you double-click the script file in `Assets`.
+
+- **Picture Editor**: Similar to the above option, used here to set when `Assets`, double-click the picture file, open the picture with the default application path.
+
+## Extension
+
+The plugin allows to directly register configuration items into preferences.The configuration items registered in the preferences will be displayed here.For details, please refer to the documentation of each plug-in.
```

Comment:
The plugin allows to directly register configuration items into preferences.The configuration items registered in the preferences will be displayed here.For details, please refer to the documentation of each plug-in.
-->
The plugin allows directly registering configuration items into preferences. The configuration items registered in the preferences will be displayed here. For details, please refer to the documentation of each plug-in.